NETWORK/NetworkAddMapObjectToSynchronisedScene.md

```diff
@@ -0,0 +1,45 @@
+---
+ns: NETWORK
+aliases: ["0x45F35C0EDC33B03B"]
+---
+## _NETWORK_ADD_MAP_OBJECT_TO_SYNCHRONISED_SCENE
+
+```c
+// 0x45F35C0EDC33B03B
+void _NETWORK_ADD_MAP_OBJECT_TO_SYNCHRONISED_SCENE(int netScene, Hash modelHash, float x, float y, float z, cs_type(float) char* animDict, char* animName, float p7, float p8, int flags);
+```
+
+Adds a map object/entity to a synchronised scene, by specifying it's hash.
+
+## Parameters
+* **netScene**: net scene id, returned by NetworkCreateSynchronisedScene
+* **modelHash**: Object model hash
+* **x**: x coord
+* **y**: y coord
+* **z**: z coord
```

Comment:
```suggestion
* **x**: X coordinate
* **y**: Y coordinate
* **z**: Z coordinate
```

Reply:
change coord to coordinate, since that is the correct spelling
some people might read "coords" instead of "coord" at a first glance